packages/liferay-npm-scripts/src/scripts/webpack.js

```diff
@@ -47,10 +47,18 @@ module.exports = function(...args) {
 };
 
 function withWebpackConfig(filename, callback) {
-	const mergeBabelLoaderOptionsPath = require.resolve(
+	let mergeBabelLoaderOptionsPath = require.resolve(
 		'../utils/mergeBabelLoaderOptions'
 	);
-	const webpackConfigPath = path.resolve(filename);
+	let webpackConfigPath = path.resolve(filename);
+
+	if (path.sep === '\\') {
```

Comment:
Isn't there a better check to do than `path.sep` to distinguish the current environment?

Reply:
Do we actually need to even check? Would a blind replacement affect *nix environments?

Reply:
Well, it's what we are replacing, the `path.sep` itself. Strictly talking, it has nothing to do with Windows vs Linux.

Reply:
And to the second question: yes, you are right. I thought it would break unix paths (because `\` is a valid posix char), but in fact it will fix them because it has to be escaped inside the quoted string no matter what it is. 

Nice catch :+1: .

Reply:
Then we should escape `\` rather than `\\`?